app/processing/genesis/transform-header.js

```diff
@@ -0,0 +1,17 @@
+const { v4: uuidv4 } = require('uuid')
+
+const transformHeader = (headerData, filename) => {
+  return {
+    correlationId: uuidv4(),
+    batch: filename,
+    invoiceNumber: headerData[1],
+    paymentRequestNumber: 1,
```

Comment:
Is this correct to be hard-coded to `1`? Does this scheme only have first payments? Should it be set as a constant?

Reply:
Yes, Genesis only produces first payments for ES.  I did consider setting it as a constant but given it's not used anywhere else and it's very clear what the property and value represent it seems a bit redundant to create files and references just for a number `1`.